packages/conf-bjack/conf-bjack.1/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
```

Comment:
it seems that this file is attempting to do two things at once. `conf-samplerate` already exists. Could you use this one and create `conf-jack` separately?